natlas-server/app/models/natlas_services.py

```diff
@@ -8,6 +10,13 @@ class NatlasServices(db.Model):
 	sha256 = db.Column(db.String(64))
 	services = db.Column(db.Text)
 
+	def __init__(self, services):
+		self.services = services
+		self.sha256 = hashlib.sha256(self.services.encode()).hexdigest()
```

Comment:
What happens if a dev changes services with `model.services = 'foobar'`?

Does this get out of date? Rails had a pre_update hook that could be used for this.